packages/@aws-cdk/aws-ec2-alpha/lib/vpc-v2-base.ts

```diff
@@ -96,6 +97,27 @@ export interface IVpcV2 extends IVpc {
    */
   readonly ipv4CidrBlock: string;
 
+  /**
+   * Optional to override inferred region
+   *
+   * @default - current stack's environment region
+   */
+  readonly region?: string;
+
+  /**
+   * The ID of the AWS account that owns the VPC
+   *
+   * @default - the account id of the parent stack
+   */
+  readonly ownerAccountId?: string;
```

Comment:
I believe these attributes should not be optional, they always have values

Reply:
fixed it in next rev